src/labkey/query/Rows.ts

```diff
@@ -22,6 +22,8 @@ export interface QueryRequestOptions extends RequestCallbackOptions {
     apiVersion?: number | string
     /** Can be used to override the audit behavior for the table the query is acting on. See [[AuditBehaviorTypes]]. */
     auditBehavior?: AuditBehaviorTypes
+    /** Can be used to provide a comment from the user that will be attached to certain detailed audit logs. */
```

Comment:
```suggestion
    /** Can be used to provide a comment from the user that will be attached to detailed audit log records. */
```

Reply:
I'm leaving the `certain` in there because not all audit logs will capture the user comments and I don't want to give the impression that that is the case.